config.toml

```diff
@@ -26,6 +26,12 @@ theme = "canvas"
       url = "/documentation/overview"
       weight = 1
       parent="documentation"
+      [[menu.main]]
+         identifier = "doc_validation"
+         name = "Validation"
+         url = "/documentation/overview/validation"
```

Comment:
I think it shouldn't be under "overview", but directly under "documentation" root instead, because reading its content it's much more than a quick overview. It's similar to what was done for runtimes monitoring (I guess you should rebase to see it)

Reply:
+1